REFRAME.md

```diff
@@ -0,0 +1,287 @@
+# ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square) Reframe

```

Comment:
You have a better name? Here's the place to pitch it, otherwise I'm less concerned about names and more concerned about specs and code and we can move on 🙃.